hq_superset/commands.py

```diff
@@ -0,0 +1,55 @@
+import requests
+import json
+
+
+def subscribe_data_sources(superset_base_url, hq_base_url, data_sources_file_path, username, apikey):
```

Comment:
Can we make this possible without the file? say if we want to do for all? I assume its already subscribed, nothing will change?

Reply:
Resolved. We would like to be explicit instead.